[2024-07-30T10:27:17.255+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T10:27:17.287+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-13T00:00:00+00:00 [queued]>
[2024-07-30T10:27:17.299+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-13T00:00:00+00:00 [queued]>
[2024-07-30T10:27:17.300+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T10:27:17.344+0000] {taskinstance.py:2330} INFO - Executing <Task(DataQualityOperator): Run_data_quality_checks> on 2018-11-13 00:00:00+00:00
[2024-07-30T10:27:17.353+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1119) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-30T10:27:17.352+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipelinev5', 'Run_data_quality_checks', 'scheduled__2018-11-13T00:00:00+00:00', '--job-id', '168', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmp3g68559c']
[2024-07-30T10:27:17.355+0000] {standard_task_runner.py:63} INFO - Started process 1120 to run task
[2024-07-30T10:27:17.355+0000] {standard_task_runner.py:91} INFO - Job 168: Subtask Run_data_quality_checks
[2024-07-30T10:27:17.468+0000] {task_command.py:426} INFO - Running <TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-13T00:00:00+00:00 [running]> on host c3b60429ae20
[2024-07-30T10:27:17.706+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipelinev5' AIRFLOW_CTX_TASK_ID='Run_data_quality_checks' AIRFLOW_CTX_EXECUTION_DATE='2018-11-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2018-11-13T00:00:00+00:00'
[2024-07-30T10:27:17.707+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T10:27:17.727+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songplays table
[2024-07-30T10:27:17.743+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:17.764+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songplays, parameters: None
[2024-07-30T10:27:17.770+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:17.773+0000] {data_quality_operator.py:30} INFO - Data quality on table songplays check passed with 2172 records
[2024-07-30T10:27:17.774+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for users table
[2024-07-30T10:27:17.791+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:17.826+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM users, parameters: None
[2024-07-30T10:27:17.836+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:17.840+0000] {data_quality_operator.py:30} INFO - Data quality on table users check passed with 80 records
[2024-07-30T10:27:17.841+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songs table
[2024-07-30T10:27:17.870+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:17.909+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songs, parameters: None
[2024-07-30T10:27:17.919+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:17.923+0000] {data_quality_operator.py:30} INFO - Data quality on table songs check passed with 71 records
[2024-07-30T10:27:17.924+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for artists table
[2024-07-30T10:27:17.955+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:17.987+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM artists, parameters: None
[2024-07-30T10:27:17.994+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:17.997+0000] {data_quality_operator.py:30} INFO - Data quality on table artists check passed with 69 records
[2024-07-30T10:27:17.999+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for times table
[2024-07-30T10:27:18.024+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:18.074+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM times, parameters: None
[2024-07-30T10:27:18.085+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:18.089+0000] {data_quality_operator.py:30} INFO - Data quality on table times check passed with 2172 records
[2024-07-30T10:27:18.091+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T10:27:18.129+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipelinev5, task_id=Run_data_quality_checks, run_id=scheduled__2018-11-13T00:00:00+00:00, execution_date=20181113T000000, start_date=20240730T102717, end_date=20240730T102718
[2024-07-30T10:27:18.213+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-30T10:27:18.263+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-30T10:27:18.316+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
